meta-mender-core/classes/mender-artifactcapsule.bbclass

```diff
@@ -0,0 +1,151 @@
+# ------------------------------ CONFIGURATION ---------------------------------
+
+# Extra arguments that should be passed to mender-artifact.
+MENDER_ARTIFACT_CAPSULE_EXTRA_ARGS ?= ""
+
+# The guid used to sign the mender update.
+MENDER_ARTIFACT_SIGNING_KEY ?= ""
+
+do_image_mender_capsule[depends] += "mender-artifact-native:do_populate_sysroot"
+
+MENDER_ARTIFACT_CAPSULE_NAME ??= "${MENDER_ARTIFACT_CAPSULE_NAME_DEFAULT}"
+MENDER_ARTIFACT_CAPSULE_NAME_DEFAULT = "uefi-capsule-${MACHINE}"
+
+MENDER_ARTIFACT_CAPSULE_NAME_DEPENDS ?= ""
+
+MENDER_ARTIFACT_CAPSULE_PROVIDES ?= ""
+
+MENDER_ARTIFACT_CAPSULE_DEPENDS ?= ""
+
+MENDER_ARTIFACT_CAPSULE_CLEAR_PROVIDES ?= ""
```

Comment:
It would be great if these could be documented [here](https://github.com/mendersoftware/mender-docs/blob/master/05.System-updates-Yocto-Project/99.Variables/docs.md)!

Reply:
There is no `master-next` branch there, should we expose those variables there, as this would be an unreleased feature?

Reply:
I created a new branch, `yocto-4.4`, you can submit the changes there!

Reply:
@kacf PR - https://github.com/mendersoftware/mender-docs/pull/2008